knative-operator/deploy/resources/kourier/kourier-v0.3.9.yaml

```diff
@@ -57,10 +57,11 @@ spec:
         app: 3scale-kourier-gateway
     spec:
       containers:
-        - args:
+        - command: ["/usr/local/bin/envoy"]
+          args:
             - -c
             - /tmp/config/envoy-bootstrap.yaml
-          image: quay.io/3scale/kourier-gateway:v0.1.3
+          image: docker.io/maistra/proxyv2-ubi8:1.0.4
```

Comment:
Should we start using 1.0.6 now? It looks like v1.0.6 is not released yet https://docs.openshift.com/container-platform/4.3/service_mesh/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-1-0-4 for production, though.